src/crawlee/_utils/requests.py

```diff
@@ -91,19 +91,23 @@ def compute_unique_key(
     *,
     keep_url_fragment: bool = False,
     use_extended_unique_key: bool = False,
+    headers: dict[str, str] | None = None,
```

Comment:
Could you please use [HttpHeaders](https://github.com/apify/crawlee-python/blob/v0.3.8/src/crawlee/_types.py#L43) type for headers? It should also help with the normalization.